SpanishFilter/sections/general_extensions.txt

```diff
@@ -2,6 +2,8 @@
 !------- JS rules ---------------------!
 !--------------------------------------!
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78702
+vermangasporno.com#%#//scriptlet("abort-on-property-read", "dataPopUnder")
```

Comment:
please change to:
`||vermangasporno.com/wp-content/cache/min/*/sdk/v*/p.js`

And add:
for newsletter form
`vermangasporno.com#?##fooside > div.col-md-4:has(> div.textwidget > form#feedburner)`

for ad banners
```
vermangasporno.com##a[target="_blank"][rel="noopener noreferrer"] > img
vermangasporno.com###mainmain
```